src/test/java/uk/gov/digital/ho/hocs/casework/api/StageResourceTest.java

```diff
@@ -141,6 +141,23 @@ public void shouldAllocateStageNull() {
         assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
     }
 
+    @Test
+    public void shouldGetActiveStagesCaseUUID() {
+        Set<Stage> stages = new HashSet<>();
+        UUID caseUUID = UUID.randomUUID();
+
+        when(stageService.getActiveStagesByCaseUUID(caseUUID)).thenReturn(stages);
+
+        ResponseEntity<GetStagesResponse> response =  stageResource.getActiveStagesByCase(caseUUID);
```

Comment:
Remove extra space after = and before stageResource
```suggestion
        ResponseEntity<GetStagesResponse> response = stageResource.getActiveStagesByCase(caseUUID);
```